airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -290,6 +291,52 @@ async def _run(self, built_connector: Container) -> StepResult:
         return StepResult(step=self, status=StepStatus.SUCCESS, stdout="Uploaded connector spec to spec cache bucket.")
 
 
+class UploadSbom(Step):
+    context: PublishConnectorContext
+    title = "Upload SBOM to metadata service bucket"
+    SBOM_KEY_PREFIX = "sbom"
+    SYFT_DOCKER_IMAGE = "anchore/syft:v1.6.0"
```

Comment:
I _think_ this is fine, but I would love us to have good discipline about updating image verisons that are referred to in this manner as a const in Python code in airbyte-ci. 

How do we make ourselves a reminder to update these at least quarterly?

Reply:
We could centralize all images in a single module with constants. I'm not sure if it's important to update this kind of image which is just a tool and does not takes part in our build process though.